src/components/Categories/MainCategoryTile.vue

```diff
@@ -40,27 +42,29 @@
 			</h4>
 		</router-link>
 	</div>
-	<div v-else-if="tileSize === 'small'" class=" tw-mb-6">
+	<div v-else-if="tileSize === 'small'" class=" tw-mb-6 tw-group">
 		<router-link
-			class="tw-no-underline
-			active:tw-no-underline visited:tw-no-underline
-			hover:tw-no-underline focus:tw-no-underline
-			tw-text-black
-			active:tw-text-black visited:tw-text-black
-			hover:tw-text-black focus:tw-text-black"
+			class="remove-link-decoration"
 			:to="`/lend-by-category/${cleanURL}`"
 			v-kv-track-event="['Lending', 'click-Category', categoryName]"
 		>
 			<div class="tw-flex">
 				<div class="tw-mr-2 tw-flex-none">
-					<kv-responsive-image class="category-image-small"
-						:images="chooseImage(tileSize)"
-						loading="lazy"
-						:alt="altText"
-					/>
+					<div class="tw-relative">
+						<kv-responsive-image class="category-image image-small"
```

Comment:
We try to stick to BEM class naming for CSS -- more info here: http://getbem.com/naming/

For this class -- it would be: `category-image category-image__small`and you can modify your css below to target the `category-image__small` class